### PR TITLE
Openjdk10 release updates

### PR DESCRIPTION
**Pull request by sxa555**
Release pipelines are failing on a section that's different between the nightly and release sections of the openjdk10 pipeline. This unifies the code between both of them

### CODE REVIEW

File: pipelines/openjdk10_release_pipeline.groovy [hunk 1]
```diff
@@ -25,8 +26,8 @@ for ( int i = 0; i < buildPlatforms.size(); i++ ) {
 				     string(name: 'PIPELINE_TIMESTAMP', value: "${PIPELINE_TIMESTAMP}")]
 		}
 		if (buildMaps[platform].test) {
-			buildMaps[platform].test.each {
-				typeTests.each {
+			stage('test') {
```
Comment (karianna) [inline pipelines/openjdk10_release_pipeline.groovy]: Adding the stage('test') seems sensible but isn't done in other scripts?

**Comment by karianna** [comment]
@sxa555 conflicts?

**Comment by karianna** [comment]
@sxa555 merge conflicts

**Comment by sxa555** [comment]
I'm going to cancel this because arm32/JDK10 is pointless and superceded by JDK11

**Pull request closed without merging**